apps/give-stark/src/app/donations/[id]/page.tsx

```
@@ -1 +1,132 @@
+            </CardContent>
+          </Card>
+        ) : null}
+        {/* META */}
+        <Card className="flex flex-col overflow-hidden max-w-full lg:max-w-[400px] xl:max-w-[500px] w-full mx-auto mt-12 lg:mt-0">
+          <CardContent className="flex flex-col pb-8 pt-3 gap-3 px-6">
+            <h1 className="mt-4 text-4xl">Donation Info</h1>
+            <div className="flex flex-col justify-start border-t mt-4 pt-4">
+              <p className="mt-4">
+                <label className="inline-block w-32 text-slate-400 font-semibold">
+                  Date:
+                </label>{' '}
```

Biome

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)